LNPopupController/LNPopupController/Private/UIViewController+LNPopupSupportPrivate.m

```diff
@@ -594,7 +594,7 @@ - (UIEdgeInsets)_vSAIFS
 		}
 		else
 		{
-			return self.popupPresentationContainerViewController.view.superview.safeAreaInsets;
+			return self.popupPresentationContainerViewController.view.safeAreaInsets;
```

Comment:
Unfortunately, this creates an issue. In the view controller scene, it miscalculates the safe area of the popup content view controller:

![Simulator Screen Shot - iPhone 11 Pro Max (iOS 14 0) - 2020-09-11 at 02 28 39](https://user-images.githubusercontent.com/2270433/92824028-99605680-f3d6-11ea-98e2-c00ac7d662ed.png)

It takes into account the popup, and it shouldn't. That's why I take the superview's safe area.

Reply:
So we could have done like that:
```objective-c
            UIEdgeInsets insets = self.popupPresentationContainerViewController.view.safeAreaInsets;
            UIEdgeInsets popupInsets = _LNPopupSafeAreas(self.popupPresentationContainerViewController);
            UIEdgeInsets finalInsets = UIEdgeInsetsMake(insets.top, insets.left, insets.bottom - popupInsets.bottom, insets.right);
            return finalInsets;
```